src/front-end/typescript/lib/pages/opportunity/code-with-us/edit/tab/addenda.tsx

```diff
@@ -97,6 +97,7 @@ const view: component_.page.View<State, InnerMsg, Route> = ({
   dispatch
 }) => {
   if (!state.opportunity || !state.addenda) return null;
+  const existingAddenda = state.opportunity.addenda;
```

Comment:
I think you can add the existing addenda to Line 47 of this file when `Addenda.init` is called. That function takes a param `existingAddenda`. This will avoid you needing to use fallback logic in the view function. Same goes for the SWU opportunity.

Reply:
complete in https://github.com/bcgov/digital_marketplace/pull/371/commits/38eae5ff0040dbb856274c5967ae37d72e8e94b8